src/foam/nanos/auth/User.js

```diff
@@ -309,7 +309,7 @@ foam.CLASS({
       section: 'personal'
     },
     {
-      class: 'foam.nanos.fs.FileProperty',
+      class: 'foam.nanos.fs.DocumentFileProperty',
```

Comment:
Is there a reason 'File' is still in the name?

Reply:
No, will rename